src/DynamoCoreWpf/Views/GuidedTour/ExitTourWindow.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Windows.Controls.Primitives;
+using Dynamo.Wpf.UI.GuidedTour;
+
+namespace Dynamo.Wpf.Views.GuidedTour
+{
+    /// <summary>
+    /// Interaction logic for ExitTourWindow.xaml
+    /// </summary>
+    public partial class ExitTourWindow : Popup
+    {
+
+        /// <summary>
+        /// This property hold a reference to the Step that was created (can be Welcome, Survey, Tooltip, ExitTour). 
+        /// </summary>
+        public Step Step { get; set; }
+
+        public ExitTourWindow(Step popupType, HostControlInfo hInfo )
+        {
+            InitializeComponent();
+
+            Step = popupType;
+
+            DataContext = Step;
+
+            PlacementTarget = hInfo.HostUIElement;
+            Placement = hInfo.PopupPlacement;
+
+            HorizontalOffset = hInfo.HorizontalPopupOffSet;
+            VerticalOffset = hInfo.VerticalPopupOffSet;         
```

Comment:
Currently the code seems very tied to step, is there a way to make it more generic. Open room for future improvement to display just any string?

Reply:
@QilongTang I've re estructured the RealTimeInfoWindow class so it won't be tied to Step and also I added a string property so the client of this class can change it so the RealTimeInfoWindow content will be updated automatically. Thanks
see commit: https://github.com/DynamoDS/Dynamo/pull/11918/commits/e6076f46422a6868c3d8e9c36106b11ada9733ec